src/charactersheet/viewmodels/common/wizard/index.js

```diff
@@ -1,377 +1,152 @@
+                value: this.dexterity,
+            },
+            {
+                name: 'Constitution',
+                shortName: 'CON',
+                value: this.constitution,
+            },
+            {
+                name: 'Intelligence',
+                shortName: 'INT',
+                value: this.intelligence,
+            },
+            {
+                name: 'Wisdom',
+                shortName: 'WIS',
+                value: this.wisdom,
+            },
+            {
+                name: 'Charisma',
+                shortName: 'CHA',
+                value: this.charisma,
             }
-        } else if (playerType.key == 'dm') {
-            // Campaign
-            actions = ['core', 'dms', 'create'];
-
-            let campaignData = self.aggregateResults()['WizardCampaignStep'];
-            params.profileImage = { type: 'email' };
-            params.playerName = campaignData.playerName;
-            params.campaign = {
-                name: campaignData.campaignName
-            };
+        ]);
+
+        // DM Fields
+
+        this.campaignName = ko.observable();
+        this.playerName = ko.observable();
+
+        //Static Data
+
+        this.raceOptions = Fixtures.profile.raceOptions;
+        this.classOptions = Fixtures.profile.classOptions;
+        this.alignmentOptions = Fixtures.profile.alignmentOptions;
+        this.backgroundOptions = Fixtures.profile.backgroundOptions;
+        this.backpackOptions = Fixtures.wizardProfile.backpackOptions;
+    }
+
+    async save() {
+        if (this.playerType() == 'character') {
+            const traits = Object.keys(DataRepository.traits).filter(key => (
+                DataRepository.traits[key].race.toLowerCase() === this.race().toLowerCase()
+            )).map((key) => ({ ...DataRepository.traits[key], tracked: null }));
+
+            const items = Object.keys(DataRepository.backpacks).filter(key => (
+                key.toLowerCase() === this.backpack().toLowerCase()
+            )).flatMap(
+                (key) => DataRepository.backpacks[key]
+            ).map(item => (
+                DataRepository.items[item.name]
+            ));
+
+            const { data } = await Hypnos.client.action({
+                keys: ['core', 'characters', 'create'],
+                params: {
+                    profile: {
+                        characterName: this.characterName(),
+                        level: this.level(),
+                    },
+                    abilityScores: ko.mapping.toJS(this.scores()),
+                    background: {
+                        name: this.background(),
+                        flaw: '',
+                        bond: '',
+                        ideal: '',
+                        personalityTrait: '',
+                    },
+                    profileImage: { type: 'email' },
+                    health: { maxHitPoints: 10 },
+                    traits: traits,
+                    items: items,
```

Comment:
ah, I can see why `age, exp, race, gender, deity, class, alignment` isn't populated. it's not being set here.